cfpb/consumerfinance.gov
# Remove Haystack code, flag references, index file, and tests from TDP

Elasticsearch7 is live, so we can remove the Haystack code, files, tests and Flag from TDP.

## How to test this PR
- Run unittest-current and check 100% coverage
```
tox -e unittest-current teachers_digital_platform
coverage report -m | egrep "Stmts|teach"
```

Check out the local activities search page (Docker setup) and make sure search and faceting work as expected.
<http://localhost:8000/consumer-tools/educator-tools/youth-financial-education/teach/activities/>
